handsontable/test/types/methods.types.ts

```diff
@@ -135,7 +135,7 @@ hot.setDataAtRowProp([[123, 'foo', 'foo'], [123, 'foo', 'foo']], 'foo');
 hot.setDataAtRowProp(123, 'foo', 'foo', 'foo');
 hot.setSourceDataAtCell([[1, 'foo', 'foo']]);
 hot.setSourceDataAtCell(123, 123, 'foo', 'sourceString');
-hot.setSourceDataAtCell(123, 123, 'foo');
+hot.setSourceDataAtCell(123, 123, 'foo', 'sourceString');
```

Comment:
Why is this change here, since it copies the above line 137? Source argument is optional, so the test, where it is not used should remain.

Reply:
Right, I edited the wrong line. I also forgot to add the test variation for an array of changes. Now tests should be complete.